packages/async-flow/src/replay-membrane.js

```diff
@@ -31,6 +34,9 @@ export const makeReplayMembrane = (
 
   let stopped = false;
 
+  const PanicTag = (template, ...args) =>
```

Comment:
Why `PanicTag` and not a simple `Panic` ?

Reply:
Done